### PR TITLE
upgrade meta_protocol_proxy dependency and update meta-protocol-control-plane-api example

### PR DESCRIPTION
**Pull request by tanjunchen**
- Update the package name aeraki/meta_protocol_proxy to meta_protocol_proxy.
- meta-protocol-control-plane-api uses the same PB with meta_protocol_proxy, adding admin package.
- Update the relevant dependency packages for meta-protocol-control-plane-api dependencies.

### CODE REVIEW

**Comment by tanjunchen** [comment]
@zhaohuabing  please review this pr, thanks.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
 # control-plane
 
-Go implementation of Meta Protocol API
+Go implementation of Meta Protocol API.
+
+The [example server](example) demonstrates how to integrate the meta-protocol-control-plane-api.
+
+## refer
```
Comment (tanjunchen) [inline README.md:7]: 我们类似 istio 依赖 https://github.com/envoyproxy/go-control-plane 实现 aeraki 与 meta-protocol-proxy 交互的 API meta-protocol-control-plane-api。
本次 PR 主要更新内容是：
1. github.com/aeraki-mesh/meta-protocol-control-plane-api  直接暴露 meta_protocol_proxy 即可，个人觉得可以不用增加父目录 aeraki。github.com/aeraki-mesh/meta-protocol-control-plane-api/aeraki/meta_protocol_proxy -> github.com/aeraki-mesh/meta-protocol-control-plane-api/meta_protocol_proxy。
2. meta-protocol-control-plane-api example 示例相关依赖包需要更新。
3. aeraki 与 meta_protocol_proxy 依赖的 PB 保持一致，便于维护。

image

@zhaohuabing  updated.

File: example/resource.go [hunk 2]
```diff
@@ -106,14 +109,11 @@ func metaProtocolRoute2HttpRoute(metaRoute metaroute.RouteConfiguration) *httpro
...
-		[]types.Resource{}, // endpoints
-		[]types.Resource{}, // clusters
-		[]types.Resource{makeRoute()},
-		[]types.Resource{}, //listeners
-		[]types.Resource{}, // runtimes
-		[]types.Resource{}, // secrets
+func GenerateSnapshot() *cache.Snapshot {
+	snap, _ := cache.NewSnapshot("1",
+		map[resource.Type][]types.Resource{
+			resource.RouteType: {makeRoute()},
```
Comment (tanjunchen) [inline example/resource.go:115]: I will write a complete example to demonstrate the meta_protocol_proxy of supporting the dubbo protocol in the future.

**Review by zhaohuabing — APPROVED** [review]
Looks great! Thanks for the refractory.

**Pull request merged**